### DISCUSSION TITLE
Update @tanstack/eslint-plugin-query to use @typescript-eslint/utils stable v8

### DISCUSSION THREAD

**Discussion by rivajunior** [TanStack/query#7967] [General]
@tanstack/eslint-plugin-query is currently setting the dependency for @typescript-eslint/utils to use v8.0.0-alpha.30.

As typescript-eslint has released the stable v8, this project could use @typescript-eslint/rule-tester and @typescript-eslint/utils with version `^8.3.0`, requiring the peerDependencies for eslint to be changed to `^8.57.0 || ^9.0.0`.

**Comment by TkDodo** [comment]
please open a PR

**Reply by rivajunior** [reply]
Done https://github.com/TanStack/query/pull/7968
